client/my-sites/checkout/upsell-nudge/index.jsx

```diff
@@ -63,7 +56,6 @@ import {
 	hasLoadedStoredCardsFromServer,
 } from 'calypso/state/stored-cards/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
-import { AnnualPlanUpgradeUpsell } from './annual-plan-upgrade-upsell';
 import { BusinessPlanUpgradeUpsell } from './business-plan-upgrade-upsell';
 import PurchaseModal from './purchase-modal';
 import { extractStoredCardMetaValue } from './purchase-modal/util';
```

Comment:
`export const ANNUAL_PLAN_UPGRADE` looks likes is have [been defined](https://github.com/Automattic/wp-calypso/pull/59771/files#diff-3283d25431f847b8456e3f5532f3e88cca8024d24dfc365def5eec8b9d7d80ddR81) for the experiment on line 76. 
Should we remove it?